### PR TITLE
chore(refs T32513): Adjust Company Name

### PR DESCRIPTION
**Pull request by salisdemos**
we renamed our company from "DEMOS E-Partizipation GmbH" to "DEMOS plan GmbH"

 **Ticket:** https://yaits.demos-deutschland.de/T32513

### CODE REVIEW

File: templates/bundles/DemosPlanUserBundle/DemosPlanUser/alternative_login.html.twig [hunk 1]
```diff
@@ -32,6 +32,7 @@
                     :header-fields="headerFields"
                     :init-items-per-page="200"
                     :table-items="rowItems"
+                    is-resizable
```
Comment (mussbach) [inline templates/bundles/DemosPlanUserBundle/DemosPlanUser/alternative_login.html.twig:35]: This has nothing to do with this PR but I assume it does not harm

**Review by mussbach — DISMISSED** [review]
lgtm, thx

File: templates/bundles/DemosPlanUserBundle/DemosPlanUser/alternative_login.html.twig (hunk 1 above)
Comment (salisdemos) [inline templates/bundles/DemosPlanUserBundle/DemosPlanUser/alternative_login.html.twig:35]:
```suggestion
```

**Review by mussbach — DISMISSED** [review]
lgtm

**Pull request merged**